tests/integration/test_data_cli.py

```diff
@@ -1051,7 +1051,7 @@ def test_asset_wait(invoke,
         runner=runner)
 
     assert not result.exception
-    assert "state: active" in result.output
+    assert "status: active" in result.output
 
```

Comment:
I would check to see if all things you want to report end up in your reporting bar. Maybe something like this? This currently fails on my laptop, since `item_type` isn't showing up.
```suggestion
assert all([item in result.output for item in [item_type, item_id, asset_type_id]])
```